src/components/organisms/CourseAgenda/_components/CourseAgendaLesson/index.tsx

```diff
@@ -89,6 +86,9 @@ export const CourseAgendaLesson: React.FC<CourseAgendaLessonProps> = ({
           />
         ))}
       </ul>
+      {!!lesson.lessons?.length && (
+        <ul className={`lesson__lessons`}>{children}</ul>
```

Comment:
```suggestion
        <ul className="lesson__lessons">{children}</ul>
```

But I would suggest using styled components here.